vue-todo/src/store/index.js

```diff
@@ -13,14 +13,20 @@ export default new Vuex.Store({
     }
   },
   mutations: {
-    ADD_TODO (state, comment) {
-      
+    ADD_TODO (state, comment) {      
       const id = state.todos.length;
+      const status = '作業中';
 
       state.todos.push({
-        id: id,
-        comment: comment,
-        status: '作業中'
+        id,
+        comment,
+        status
+      });
+    },
+    DELETE_TODO (state, id) {
```

Comment:
関数名に関して
前回ご指摘すれば良かったのですが、一般的には先頭小文字からのキャメルケースを使用することが多いです。
動作としては問題ないですが、修正しておくと良いと思います。
```suggestion
    deleteTodo (state, id) {
```